component/common/src/main/java/org/wso2/carbon/identity/smsotp/common/constant/Constants.java

```diff
@@ -71,6 +73,8 @@
                 "No OTP found for the user Id : %s."),
         CLIENT_SLOW_DOWN_RESEND("SMS-60010", "Slow down.",
                 "Please wait %s seconds before retrying."),
+        CLIENT_OTP_VALIDATION_BLOCKED("SMS-60011", "Maximum no of failed validation attempted exceeded.",
+                "Maximum no of failed validation attempted exceeded. User id : %s."),
```

Comment:
`attempted exceeded for User id: %s.` WDYT?

Reply:
Fixed with 13c0ee012c419920360c82927c06d3f0d20ad8c8